keyboards/planck/keymaps/unagi/config.h

```diff
@@ -0,0 +1,46 @@
+#ifndef CONFIG_USER_H
```

Comment:
Could you replace the boilerplate?  

Specifically, could you replace: 
```c
#ifndef CONFIG_USER_H
#define CONFIG_USER_H

#include "config_common.h"
```
And the `#endif` at the end with just `#pragma once`?